settings/auth.go

```diff
@@ -12,17 +12,27 @@ const (
 	EnvKeycloakRealm        = "KEYCLOAK_REALM"
 	EnvKeycloakClientID     = "KEYCLOAK_CLIENT_ID"
 	EnvKeycloakClientSecret = "KEYCLOAK_CLIENT_SECRET"
+	EnvKeycloakAdminUser    = "KEYCLOAK_ADMIN_USER"
+	EnvKeycloakAdminPass    = "KEYCLOAK_ADMIN_PASS"
+	EnvKeycloakAdminRealm   = "KEYCLOAK_ADMIN_REALM"
 	EnvAddonToken           = "ADDON_TOKEN"
+	EnvRolePath             = "ROLE_PATH"
+	EnvUserPath             = "USER_PATH"
 )
 
 //
 // Defaults
 const (
 	KeycloakHost         = "https://localhost:8081"
-	KeycloakRealm        = "tackle"
-	KeycloakClientID     = "tackle-hub"
-	KeycloakClientSecret = "tackle"
-	AddonToken           = "tackle"
+	KeycloakRealm        = "konveyor"
+	KeycloakClientID     = "konveyor"
+	KeycloakClientSecret = "konveyor"
+	AddonToken           = "konveyor"
+	KeycloakAdminUser    = "admin"
+	KeycloakAdminPass    = "admin"
+	KeycloakAdminRealm   = "master"
+	RolePath             = "/tmp/roles.yaml"
+	UserPath             = "/tmp/users.yaml"
 )
```

Comment:
In other settings, we don't have const for defaults but define them in the load() instead.
The thinking here is that we don't outside code to accedentally (or intentionally) use the default instead of the _Setting_.
I realize you did not introduce the _const_ in the PR but would recommend it would be a good opportunity to address it.